index.html

```diff
@@ -1,51 +1,306 @@
+          </ul>
+        </div>
+        <div class="askill">
+          <div class="askilltext">
+            <div class="arectangle1"></div>
+            <h3 class="askilltitle">Frameworks</h3>
+          </div>
+          <ul class="askillist">
+            <li class="askillitem">
+              <p class="askillitemtext">Bootstrap</p>
+            </li>
+            <li class="askillitem">
+              <p class="askillitemtext">Ruby on Rails</p>
+            </li>
+            <li class="askillitem">
+              <p class="askillitemtext">RSPec</p>
+            </li>
+            <li class="askillitem">
+              <p class="askillitemtext">CapyBara</p>
+            </li>
+            <li class="askillitem">
+              <p class="askillitemtext">Selenium</p>
+            </li>
+          </ul>
+        </div>
+        <div class="askill">
+          <div class="askilltext">
+            <div class="arectangle2"></div>
+            <h3 class="askilltitle">Skills</h3>
+          </div>
+          <ul class="askillist">
+            <li class="askillitem">
+              <p class="askillitemtext">Codekit</p>
+            </li>
+            <li class="askillitem">
+              <p class="askillitemtext">Github</p>
+            </li>
+            <li class="askillitem">
+              <p class="askillitemtext">Codepen</p>
+            </li>
+            <li class="askillitem">
+              <p class="askillitemtext">Gitlab</p>
+            </li>
+            <li class="askillitem">
+              <p class="askillitemtext">Terminal</p>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section class="contact">
+      <div class="ctitle">
+        <h3 class="ctitletext">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+      </div>
+      <form class="contactform">
+        <input type="text" name="name" placeholder="Full name" class="cdata" />
+        <input
+          type="text"
+          name="email"
+          placeholder="Email address"
+          class="cdata"
+        />
+        <textarea
+          name="message"
+          placeholder="Enter Text Here"
+          class="cmessage"
+        ></textarea>
+        <input class="csubmit" type="submit" value="Get in touch" />
+      </form>
+      <ul class="csocial">
+        <li class="csocialimg">
+          <img src="./Images/GithubDark.png" alt="" />
+        </li>
+        <li class="csocialimg">
+          <img src="./Images/linkedinDark.png" alt="" />
+        </li>
+        <li class="csocialimg">
+          <img src="./Images/HandDark.png" alt="" />
+        </li>
+        <li class="csocialimg">
+          <img src="./Images/TwiterDark.png" alt="" />
+        </li>
+        <li class="csocialimg">
+          <img src="./Images/MediunDark.png" alt="" />
+        </li>
+      </ul>
+      <div class="fmargin"></div>
+    </section>
```

Comment:
- Please wrap your `<img>` inside of an `<a>` tag as it will be a link to your social handles. like below
```html
 <li class="socialimg">
   <a href ="#"> <img src="./Images/Icon-Medium.png" alt="" /> </a>
  </li>
```

If you find this feedback confusing please reach pout to me on Slack `@kanu mike chibundu`